salalib/axialmap.cpp

```diff
@@ -1350,31 +1353,35 @@ int ShapeGraphs::convertDataToAxial(Communicator *comm, const std::string& name,
    ShapeGraph& usermap = tail();
 
    usermap.init(lines.size(),region);  // used to be double density
+   usermap.initialiseAttributesAxial();
+   std::map<int, float> extraAttributes;
+   std::vector<int> columns;
+   if (copydata)   {
+       AttributeTable& input = shapemap.getAttributeTable();
+       AttributeTable& output = usermap.getAttributeTable();
+       for (int i = 0; i < input.getColumnCount(); i++) {
+          std::string colname = input.getColumnName(i);
+          for (size_t k = 1; output.getColumnIndex(colname) != -1; k++){
+             colname = dXstring::formatString((int)k,input.getColumnName(i) + " copy %d");

```

Comment:
This "copy" at the end of the column name is not applied in the other changes in this commit. Shouldn't it be the same in all?